source/extensions/filters/udp/udp_proxy/router/router.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "envoy/common/pure.h"
+#include "envoy/network/address.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace UdpFilters {
+namespace UdpProxy {
+namespace Router {
+
+/**
+ * The router.
+ */
+class Router {
+public:
+  virtual ~Router() = default;
+
+  virtual const std::string route(Network::Address::InstanceConstSharedPtr address) const PURE;
```

Comment:
Is "address" a source or a destination address? (Presumably source. If so source_address would be a nice name for this parameter).

What's the return value? Presumably a cluster name? It'd be nice to add a comment to this effect.

Reply:
Ok.

Reply:
Let's rename "address" to "source address" in the parameter. (Here and elsewhere)

Reply:
I will change these names.